docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+This link should bring you to the AWS Console and home page.
+
+    Once here, be sure that you are in the `us-east-2` region, by selecting the drop-down menu in the tool bar.
+
+    <figure markdown="span">
+        ![Select region](../../img/creating-vcs-1.png){width="400"}
+    </figure>
+
+1. You will need to use the AWS Service Catalog to create virtual computers.
+Open the AWS Service Catalog by using the search bar and typing, `service catalog`.
+
+    <figure markdown="span">
+        ![Select service catalog](../../img/creating-vcs-2.png){width="600"}
+    </figure>
+
+1. Select `LightsailInstance` in the product list, and click `Launch product`.
+
+    <figure markdown="span">
+        ![Launch product](../../img/creating-vcs-3.png){width="600"}
+    </figure>
+
+1. You will then choose the name and configurations for your virtual computer.
+
+    - Start by providing a `Provisioned product name`.
+    <!--TODO Do we want to provide guidance on names?-->
+    - Pick an `Application` to use from the drop-down menu.
+    This installs any additional applications along with the pre-installed applications.
+        - If you are planning to develop in R, we recommend choosing `Rstudio`, which includes RStudio.
+        - If you are planning to develop in Python, we recommend choosing `VSCodium`, which includes VSCodium, a distribution of Microsoft's editor VS Code.
+        - For information on all application options, see [the LSfR documentation on Applications](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html).
+    - Choose any of the availability zones.
+    - Name your instance, be sure to make sure this is different than the name chose for your product.
+    - Pick the size for your instance.
+        - Use this [table describing the total vCPUs, memory, and storage space](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html#plans) to choose the most appropriate instance plan.
+        - Before you request a plan with GPUs, you will need to put in a [request with the Data Lab team](../../getting-started/accessing-resources/getting-access-to-compute.md#gpu-instance-access).
+    - Set the `ShutdownIdlePercent` to 5 and the `ShutdownTimePeriod` to 10.
+    This means if the instance is using less than 5% of the total available CPUs or inactive for more than 10 minutes, the instance will temporarily shut down.
+    This _does not_ terminate your instance, it temporarily stops any idle instances until you are ready to resume your work.
+
+    <figure markdown="span">
+        ![Configure instance](../../img/creating-vcs-4.png){width="600"}
+    </figure>
+
+1. Once you have configured your instance, click `Launch product`.
+You have now created a virtual computer!
+
+## How to access a virtual computer
+
+Once you have created your virtual computer, you will be able to access and launch the computer on a web browser.
+Note that it can take 5-10 minutes to create the instance.
```

Comment:
In my experience, you can access an instance before GitKraken and R installation is completed. I think you might have encountered that in testing based on your notes.
```suggestion
We recommend waiting ~10 minutes before accessing the instance.
It takes time create the instance and for all software to be installed.
```